initializr-generator-spring/src/main/java/io/spring/initializr/generator/spring/scm/git/GitProjectGenerationConfiguration.java

```diff
@@ -62,6 +62,7 @@ public GitIgnoreCustomizer gradleGitIgnoreCustomizer() {
 			gitIgnore.getGeneral().add(".gradle", "build/", "!gradle/wrapper/gradle-wrapper.jar",
 					"!**/src/main/**/build/", "!**/src/test/**/build/");
 			gitIgnore.getIntellijIdea().add("out/", "!**/src/main/**/out/", "!**/src/test/**/out/");
+			gitIgnore.getSts().add("bin/");
```

Comment:
I think we should add `"!**/src/main/**/bin/"` and `"!**/src/test/**/and/"` as well here so that any directories named `bin` beneath `src/main` and `src/test` aren't ignored.

Reply:
Good idea, I've added them